src/main/java/org/opensearch/security/securityconf/impl/v7/InternalUserV7.java

```diff
@@ -154,5 +153,11 @@ public void setStatic(boolean _static) {
             this._static = _static;
         }
         
-        
+        public String getSaved_tenant(){
+            return saved_tenant;
+        }
+
+        public void setSaved_tenant(String s){
+            saved_tenant = s;
+        }
     }
```

Comment:
Please add an empty line at the end of the file.

Reply:
Changed

Reply:
I dont see an empty line here. Am I missing something ?

Reply:
sorry; just added it

Reply:
added